class-gravity-flow.php

```diff
@@ -5989,6 +5989,12 @@ public function filter_after_update_entry( $form, $entry_id ) {
 		 * @param array $menu The current WP Dashboard Menu.
 		 */		
 		public function show_inbox_count( $menu ) {
+
+			$show = apply_filters( 'gravityflow_inbox_count_display', $show );
+			if ( isset( $show ) && ! $show ) {
```

Comment:
We don't usually use an approach like this when adding filters. Normally the argument would be defaulted to `true` or `false` e.g.

```suggestion
			$show = apply_filters( 'gravityflow_inbox_count_display', true );
			if ( ! $show ) {
```

There should also be a docblock with since and param tags.